pkg/codegen/report/report.go

```diff
@@ -205,7 +207,7 @@ func (r *reporter) Close() error {
 func (r *reporter) DefaultExport() error {
 	r.m.Lock()
 	defer r.m.Unlock()
-	dir, ok := os.LookupEnv(ExportTargetDir)
+	dir, ok := ExportTargetDir.Underlying()
```

Comment:
I suppose this should eventually migrate to `env.Global().ExportTargetDir().Underlying()`?
Just wondering

Reply:
`Value.Underlying()` retrieves from the global store by default. I don't see the need to attach it to an explicit `env.Env` unless the `env.Env` is injected (and thus dynamic).

Reply:
Perfect, sounds good!